share/github-backup-utils/ghe-backup-config

```diff
@@ -22,6 +22,10 @@ BACKUP_UTILS_VERSION="$(cat $GHE_BACKUP_ROOT/share/github-backup-utils/version)"
 
 # Add the bin and share/github-backup-utils dirs to PATH
 PATH="$GHE_BACKUP_ROOT/bin:$GHE_BACKUP_ROOT/share/github-backup-utils:$PATH"
+# In Solaris 11 reassure that gnu utils are used and add /usr/sbin for unlink
+if [ "SunOS 5.11" = "$(uname -sr)" ]; then
+  PATH="/usr/gnu/bin:$PATH:/usr/sbin"
+fi
```

Comment:
I agree 100%.  The point I was trying to make is you don't need to install any of the GNU pkgs to get a fully functional Solaris 11 system.  If you so desire, you can install the GNU utils from source into `/usr/local/bin` or the old traditional location for third party software of `/opt` instead. In these cases `/usr/gnu/bin` wouldn't be a valid path.